### PR TITLE
Added more fun commands from Dyno

### PR DESCRIPTION
**Pull request by H4ckerJ4cker**
Might as well add these commands to the bot. Commands added:

- :tryharder
- :neveragoodtime
- :hac and :dox
- :beano and more fun commands
- text and reaction responses to words in messages

Let me know if there any fun commands or reactions that need added, it can also react to a word with emojify.

### CODE REVIEW

**Comment by Rowanyeet** [comment]
why wouldn't we want whosthatalt?

**Review by 404dcd — APPROVED** [review]

**Review by Sh3llcod3 — APPROVED** [review]
What about `:hac` and `:dox` []()

**Comment by H4ckerJ4cker** [comment]
@Sh3llcod3 .hac and .dox are regex. Should I add it as regex or add it as a command?

**Comment by Sh3llcod3** [comment]
Welp, as a command maybe?

File: cdbot/cogs/fun.py [hunk 1]
```diff
@@ -577,6 +577,28 @@ async def baldj(self, ctx: Context, *, text: str):
...
+        await ctx.send("https://www.youtube.com/watch?v=t-bgRQfeW64")
+
+    @command()
+    async def whosthatalt(self, ctx: Context):
+        """
+        Returns the "WHO'S THAT ALT" image.
+        """
+        await ctx.send("**WHO'S THAT ALT?!**")
+        await ctx.send("https://cdn.discordapp.com/attachments/411573884597436416/767093622897639434/CveO6GXh.jpg")
+
```
Comment (thebeanogamer) [inline cdbot/cogs/fun.py:601]: This isn't helpful and shouldn't come back.

```suggestion

```

**Review by thebeanogamer — CHANGES_REQUESTED** [review]

File: cdbot/cogs/fun.py (hunk 1 above)
Comment (Sh3llcod3) [reply cdbot/cogs/fun.py]: Yep agreed

File: cdbot/cogs/fun.py [hunk 2]
```diff
@@ -238,6 +238,11 @@ def check(reaction, user):
             )
         ):
             await message.add_reaction("\N{WAVING HAND SIGN}")
+        # Simple regex responses.
+        if ".hac" in message.content.lower():
+            await message.channel.send("Charging the Low Orbit Ion Canon, please stand by!")
+        elif ".dox" in message.content.lower():
+            await message.channel.send("OK, scraping their parent's public Facebook feed!")
```
Comment (thebeanogamer) [inline cdbot/cogs/fun.py:245]: This isn't a regex and it'll be very greedy and fire a lot of the time it shouldn't. Why can't they just be commands like everything else

**Review by thebeanogamer — CHANGES_REQUESTED** [review]

**Comment by H4ckerJ4cker** [comment]
needs some work

**Comment by H4ckerJ4cker** [comment]
everything works here except .beano probs cuz idk regex well. any suggestions welcome

**Review by thebeanogamer — APPROVED** [review]

**Pull request merged**